docs/articles/expensify-classic/connect-credit-cards/company-cards/Troubleshooting.md

```diff
@@ -3,97 +3,156 @@ title: Troubleshooting
+2. **Update company cards.** Go to **Settings** > **Domains** > _[Domain Name]_ > **Company Cards**. Click on the card in question and select **Update** to refresh the card feed.
+3. **Reconcile cards.** Navigate to the **Reconciliation** section under **Settings** > **Domains** > _[Domain Name]_ > **Company Cards**. Refer to the detailed guide on how to use the [Reconciliation Dashboard](https://help.expensify.com/articles/expensify-classic/connect-credit-cards/company-cards/Reconciliation#identifying-outstanding-unapproved-expenses-using-the-reconciliation-dashboard).
+4. **Review transactions.** Use the [Reconciliation Dashboard](https://help.expensify.com/articles/expensify-classic/connect-credit-cards/company-cards/Reconciliation#identifying-outstanding-unapproved-expenses-using-the-reconciliation-dashboard) to view all transactions within a specific timeframe. Transactions will display on the **Expenses** page based on their posted date. If needed, uncheck the Use Posted Date checkbox near the filters to view transactions based on their Transaction Date instead.
+5. **Address gaps.** If there is a significant gap in transactions or if transactions are still missing, contact Concierge or your Account Manager. They can initiate a historical data update on your card feed to ensure all transactions are properly imported.
+
+# General troubleshooting 
+
+## Common import problems 
+
+If company cards seem to be disconnected or not working as expected, troubleshoot by: 
+- Clicking **Update Card** under:
+  - **Settings** > **Account** > **Credit Card Import** for personal cards, or
+  - **Settings** > **Domains** > _[Domain Name]_ > **Company Cards** for company cards.
+- If a **Fix** option appears, click on it and follow the steps to fix the connection.
+
+## Alternative workarounds
+For persistent issues with API-based connections (e.g., American Express, Chase, Wells Fargo), the alternative option is to [manually add expenses](https://help.expensify.com/articles/expensify-classic/expenses/expenses/Add-an-expense), or [upload expenses via CSV](https://help.expensify.com/articles/expensify-classic/bank-accounts-and-credit-cards/company-cards/CSV-Import).
+
+## The connection is established but there are no cards to assign
+When establishing the connection, you must assign cards during the same session. It isn't possible to create the connection, log out, and assign the cards later, as the connection will not stick and will require you to reattempt the connection.
+
+# Addressing duplicate expenses 
+
+If a workspace member is experiencing duplicated expenses, this is typically due to:
+
+ -  A cardholder having accidentally imported the card as a personal credit card, in addition to being assigned the company card by a Domain Admin.  
+   - To troubleshoot, have the employee navigate to **Settings** > **Account** > **Credit Card Import** and confirm that their card is only listed once. 
+
```

Comment:
NAB: Let's remove this blank line please

![image](https://github.com/user-attachments/assets/69652cca-ef72-4fd8-894f-80b711eef526)